modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/components/ExperienceItem.js

```diff
@@ -122,13 +122,22 @@ const ExperienceItem = ({
 											<ExperienceLockIcon />
 										)}
 
-										{experience.active && (
+										{experience.active ? (
 											<ClayLabel
-												className="inline-item-after"
+												className="flex-shrink-0 inline-item-after"
```

Comment:
Maybe this commit fixing the length of the experience name can be fixed up with `dc73769: LPS-165312 Display inactive label in experience selector` ?

Reply:
Since it was a bug already present (for the active label only) I preferred to include it in a separate commit.